docs: fix typo in `save-peer` description

chalkygames123 opened the pull request:
Corrected the typo and then ran `TAP_SNAPSHOT=1 npm test`.

wraithgar commented on tap-snapshots/test/lib/commands/view.js.test.cjs:
```diff
@@ -82,7 +82,7 @@ dist
 dist-tags:
 [1m[32mlatest[39m[22m: 1.0.0
 
-published [33myesterday[39m
```
not sure why this changed ended up in your tests but it's making CI fail. Likely just need to revert this one to match`release-next`, and it's something for us to look into.

wraithgar commented:
Try running `npm run snap` instead, there's a `postsnap` hook that runs that rebuilds additional files that are dependent on these config definitions.

chalkygames123 commented:
Okay, I'll do. Thanks for your instruction :)

chalkygames123 commented:
Hmm, while the changes now include `config.md`, CI still fails. Should I only commit the change in `definitions.js` for now?

wraithgar commented:
I think if you omit the changes in ` tap-snapshots/test/lib/commands/view.js.test.cjs` you should be ok. If not we'll just call it ok and I'll fix it when I merge it and make sure it's passing.

chalkygames123 committed:
docs: fix typo in `save-peer` description

PR-URL: https://github.com/npm/cli/pull/4072
Credit: @chalkygames123
Close: #4072
Reviewed-by: @wraithgar

chalkygames123 commented:
Now it looks ok!

wraithgar commented:
Thanks for sticking with it, I know these tests are tricky to get passing but they save us a lot of headache down the line when it comes to cutting our releases.
